git-hg/updatetags8.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+#
+# ===========================================================================
+# (c) Copyright IBM Corp. 2018 All Rights Reserved
+# ===========================================================================
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+#
+# ===========================================================================
+
+if [ $# -lt 2 ]; then
+   echo Usage: $0 '[jdk8u|jdk9] TAGS'
+   echo Version supplied should match a reposotiry in adoptopenjdk/openjdk-VERSION
+   echo And also http://hg.openjdk.java.net/VERSION/VERSION
+   exit 1
+fi
+[ -z "$WORKSPACE" -o ! -d "$WORKSPACE" ] && echo Cannot access \$WORKSPACE: $WORKSPACE && exit 2
+
+set -euo pipefail
+
+MODULES=(corba langtools jaxp jaxws nashorn jdk hotspot)
+
+# These variables needs to define in Jenkin's job pipeline
+OPENJDK_VERSION="$1"
+TAGS="$2"
```

Comment:
Hmm, actually wondering if we should default to all tags and make $2 an optional arg

Reply:
If you use all tags (there are hundreds) then it'll take weeks and probably wouldn't be particularly useful. If I was going to run it from scratch I'd probably manually set it to be all the quarterly release tags (which was what I was initially planning to do when repopulating the repository). As a default I'd be a but more incluned to go for "the latest one" which I think is what our previous version of the script did.